web/src/elements/buttons/ActionButton/ak-action-button.stories.ts

```diff
@@ -10,10 +10,26 @@ import { html, TemplateResult } from "lit";
 const metadata: Meta<AKActionButton> = {
     title: "Elements / <ak-action-button>",
     component: "ak-action-button",
+    tags: ["autodocs"],
     parameters: {
         docs: {
             description: {
-                component: "A four-state button for asynchronous operations",
+                component: `
```

Comment:
Adds syntax highlighting

```suggestion
                component: /* md */ `
```

<img width="761" height="370" alt="Screenshot 2025-07-21 at 20 13 48" src="https://github.com/user-attachments/assets/7ffba284-4b27-46e5-bd0c-5f3df8efcc68" />